app/config/wallabag.yml

```diff
@@ -183,3 +183,4 @@ parameters:
         - 'text/html'
         - 'application/vnd.ms-excel'
     wallabag.resource_dir: "%kernel.project_dir%/web/uploads/import"
+    wallabag.user_agent: "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/535.2 (KHTML, like Gecko) Chrome/15.0.874.92 Safari/535.2"
```

Comment:
Can you define that value in `app/config/parameters.yml.dist` so that admin user will be able to change it without touching the code.

Reply:
It's done, let me know if you prefer a different name for the parameter.